examples/cube/main.rs

```diff
@@ -167,9 +166,9 @@ impl<B: gfx::Backend> gfx_app::Application<B> for App<B> {
         }
     }
 
-    fn render<Gp>(&mut self, (frame, semaphore): (gfx_core::Frame, &<B::Resources as gfx::Resources>::Semaphore),
-                  pool: &mut Gp, queue: &mut gfx_core::queue::GraphicsQueueMut<B>)
-        where Gp: gfx_core::GraphicsCommandPool<B>
+    fn render<Gp>(&mut self, (frame, semaphore): (gfx::Frame, &<B::Resources as gfx::Resources>::Semaphore),
```

Comment:
hopefully, the semaphore will eventually become something like `handle::Semaphore<B::Resources>`